Applications/spire/include/spire/book_view/book_view_window.hpp

```diff
@@ -63,20 +63,27 @@ namespace Spire {
       mutable ClosedSignal m_closed_signal;
       BookViewProperties m_properties;
       SecurityInputModel* m_input_model;
+      std::shared_ptr<BookViewModel> m_model;
       SecurityStack m_securities;
       Nexus::Security m_current_security;
       QWidget* m_body;
       QVBoxLayout* m_layout;
       TechnicalsPanel* m_header_widget;
```

Comment:
Rename this from m_header_widget to m_technicals_panel. You can learn to use Visual Studio's renaming feature to do this safely.